pyproject.toml

```diff
@@ -13,12 +13,22 @@ dependencies = ["bsb-core~=4.0"]
 
 [project.optional-dependencies]
 test = ["bsb-core[parallel]", "bsb-hdf5~=4.0", "coverage~=7.0"]
+dev = [
+    "bsb-test[test]",
+    "build~=1.0",
+    "twine~=4.0",
+    "pre-commit~=3.5",
+    "black~=24.1.1",
+    "isort~=5.12",
+    "snakeviz~=2.1",
+    "bump-my-version~=0.24"
+]
 
 [tool.flit.module]
 name = "bsb_test"
 
 [tool.black]
-line-length=120
+line-length = 90
```

Comment:
why change this?

Reply:
I wanted to match `bsb-core` black configuration. I can revert this, if you prefer to have different configs for each bsb repos.